ginga/rv/plugins/Catalogs.py

```diff
@@ -372,26 +401,38 @@ def redo(self):
             return True
 
         # Copy the image parameters out to the widget
-        d = { 'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
-              'height': ht, 'r': radius, 'r2': radius,
-              'r1': 0.0,
-              }
+        d = {'ra': ra_ctr, 'dec': dec_ctr, 'width': str(wd),
+             'height': ht, 'r': radius, 'r2': radius,
+             'r1': 0.0}
         self._update_widgets(d)
         return True
 
-    def btndown(self, canvas, event, data_x, data_y):
+    def btndown(self, canvas, event, data_x, data_y, viewer):
         return True
 
-    def btnup(self, canvas, event, data_x, data_y):
-
-        objs = self.canvas.get_items_at(data_x, data_y)
+    def btnup(self, canvas, event, data_x, data_y, viewer):
```

Comment:
This is rather a pronounced change which could possibly impact performance.  Before, the canvas is queried at the point of clicking to determine which objects are there.  The number of items returned is likely to be small.  But in the PR, the number of items returned may be large (e.g. in our case hundreds of stars circled) and then each one is tested for inclusion of the point of clicking.

Any particular reason for this change?